manifests/base/jupyter-habana-notebook-imagestream.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com//opendatahub-io/notebooks/tree/main/habana"
+    opendatahub.io/notebook-image-name: "HabanaAI"
+    opendatahub.io/notebook-image-desc: "Jupyter notebook image with a set of habana libraries that advanced AI/ML notebooks will use as a base image to provide a standard for libraries avialable in all notebooks"
+    opendatahub.io/notebook-image-order: "7"
+    opendatahub.io/recommended-accelerators: '["habana.ai/gaudi"]'
+  name: jupyter-habana-notebook
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  # 1.11.0 Version of the image n
+  - annotations:
+      opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.8"},{"name":"Habana","version":"1.11.0.587"}]'
+      opendatahub.io/notebook-python-dependencies: '[{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.23"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Tensorflow","version":"2.12.1"},{"name":"PyTorch","version":"2.0.1"},{"name":"Elyra","version":"3.15"}]'
+      openshift.io/imported-from: quay.io/opendatahub/workbench-images
+    from:
+      kind: DockerImage
+      name: $(odh-habana-notebook-image-n)
+    name: "1.11.0"
```

Comment:
I am not sure about the `name:` as 1.11.0. IMO It should comply with the rest of the notebook's naming. I can see that is not the same case. Maybe we should discuss this with the UI folks

Reply:
As you suggested , i will include this question for UX folks.
for now lets proceed with these names, as 2023.a and 1.2 , will not help out here.

Reply:
Sounds good to me